nautobot/extras/tests/test_models.py

```diff
@@ -376,6 +376,11 @@ def test_dynamic_group_assignment_uniqueness(self):
             device_role=self.devicerole,
             device_type=self.devicetype,
         )
+
+        # Ensure group membership cache is up to date
+        self.dynamic_groups.get_members(skip_cache=False, force_update_cache=True)
+        self.dynamic_group_2.get_members(skip_cache=False, force_update_cache=True)
```

Comment:
Would it be more appropriate to override_settings so that caching of members is disabled?